src/syncerd/monero_syncer.rs

```diff
@@ -224,12 +224,19 @@ async fn sweep_address(
     spend: monero::PrivateKey,
     network: &monero::Network,
     wallet_mutex: Arc<Mutex<monero_rpc::WalletClient>>,
+    from_height: Option<u64>,
 ) -> Result<Vec<Vec<u8>>, Error> {
     let keypair = monero::KeyPair { view, spend };
     let password = s!(" ");
     let address = monero::Address::from_keypair(*network, &keypair);
     let wallet_filename = format!("sweep:{}", address);
 
+    // add some extra leeway for the wallet restore height
+    let restore_height = match from_height {
+        Some(height) if height > 10 => Some(height - 10),
+        _ => None,
+    };
+
```

Comment:
2nd time: this gives too many degrees of freedom

Reply:
Let me test again without it and then drop it. Kind of weird to have this in the first place.

Reply:
Dropped in fa8a433443e39f15c753f598c8ae704b06a5d4d1